kibana-reports/public/components/context_menu/context_menu.js

```diff
@@ -137,7 +137,12 @@ const generateInContextReport = (
         $('#reportGenerationProgressModal').remove();
         addSuccessOrFailureToast('success');
       } else {
-        addSuccessOrFailureToast('failure');
+        if (response.status === 403) {
```

Comment:
Is 403 the only statusCode for permission? Maybe confirm with @akbhatta

Reply:
I think it is but @akbhatta  can you confirm?

Reply:
yes. All permission-deny mapped to FORBIDDEN(403)